docs/06.REFERENCE-COUNTING.md

```diff
@@ -48,16 +48,17 @@ following code is an **INCORRECT WAY** of releasing the 3.14 value.
 
 JerryScript API functions returning with a `jerry_value_t` always
 return with a new live reference. Passing a `jerry_value_t` to
-an API function never releases its reference. The next example
-shows this behaviour through property getting and setting.
+an API function never releases its reference (except for
+`jerry_release_value`, obviously). The next example shows this
+behaviour through property getting and setting.
```

Comment:
Feel free to change it, but emphasize that you can count on this, unless explicitly stated otherwise (at this moment there is otherwise).